parm/config/config.anal

```diff
@@ -93,9 +104,17 @@ if [ $REALTIME = "NO" ]; then
     fi
 fi
 
-if [ $LEVS = "128" ]; then
-   export GRIDOPTS="nlayers(63)=1,nlayers(64)=1,"
-   export SETUP="gpstop=55,nsig_ext=56,$SETUP"
+# TEST - use experimental dumps
```

Comment:
Will need to remember to remove pointers to x-dumps or wrap in a date if-block so the experimental versions are used in the future for dates before these files went into production.

Reply:
good point.  This is copied over from v16rt2. Need to check when and for how long these two have been used in the parallel.   For now, my purpose if to make sure this branch can reproduce v16rt2

Reply:
Please contact Haixia to get start dates for AHI and ABI.